src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <nav class="nav">
+        <a href="./index.html" 
+          class="logo-link" 
+          target="_blank">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <ul class="nav_link">
+          <a href="#" class="refer is-active" 
+            target="_blank">Apple
+          </a>
```

Comment:
```suggestion
        <ul class="nav__link">
          <li class="nav__item">
            <a 
              href="#"
              class="refer is-active" 
              arget="_blank">
                Apple
            </a>
```
You need to add `li` for each link and please read the checklist and pay attention to the checklist

Reply:
Also, pay attention to the class names.